lib/ubersicht/ingestion/client.rb

```diff
@@ -28,6 +28,20 @@ def initialize(options = {}, &block) # rubocop:disable Metrics/AbcSize, Metrics/
         @conn = setup_conn(url, user, pass, &block)
       end
 
+      def ingest(transaction_type, event_code, event_date, payload = {})
```

Comment:
What do you think about a default value for the date? i.e. use the current time in case it's not passed?

Reply:
We can do that for sure. If time is not required option (because of default value), what do you think about moving it to event payload? We did the same with transaction_id.